src/scripts/main.js

```diff
@@ -2,7 +2,19 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
+let count = 1;
 
-// write your code here
+people.forEach(person => {
+  table.insertRow();
+
+  const row = document.querySelectorAll('tr');
+
+  row[count].insertCell(0).innerText = person.name;
+  row[count].insertCell(1).innerText = person.sex === 'm' ? 'Male' : 'Female';
+  row[count].insertCell(2).innerText = person.born;
+  row[count].insertCell(3).innerText = person.died;
+  row[count].insertCell(4).innerText = person.died - person.born;
+  row[count].insertCell(5).innerText = Math.ceil(person.died / 100);
+  count++;
```

Comment:
you can use the second parametr of callback which you pass to the forEach instead of counter